### ISSUE TITLE
Unsure about correct alignment of block ends

### ISSUE THREAD

**Issue by lawliet89** [rubocop/rubocop#447]
Given the following code:

``` ruby
    params = default_options.merge(options)
              .delete_if { |k, v| v.nil? }
              .each_with_object({}) do |(k, v), new_hash|
                new_hash[k.to_s] = v.to_s
              end
```

Rubocop complains with the `BlockAlignment` cop and wants me to align it as such:

``` ruby
    params = default_options.merge(options)
              .delete_if { |k, v| v.nil? }
              .each_with_object({}) do |(k, v), new_hash|
                new_hash[k.to_s] = v.to_s
    end
```

Is this intended behaviour? I find it's not actually better style to align it as such.
